src/js/common/helpers/login-helpers.js

```diff
@@ -111,13 +99,13 @@ export function getLoginUrls(onUpdateLoginUrls) {
   return loginUrlsRequest;
 }
 
-export function handleLogin(loginUrl, onUpdateLoginUrl) {
+export function handleLogin(loginUrl) {
   window.dataLayer.push({ event: 'login-link-clicked' });
   if (loginUrl) {
     window.dataLayer.push({ event: 'login-link-opened' });
     const receiver = window.open(`${loginUrl}&op=signin`, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
     receiver.focus();
-    return getLoginUrl(onUpdateLoginUrl);
+    return null;
```

Comment:
Should this not be `return getLoginUrls(onUpdateLoginUrls)`? This is a change that's causing some unit test failures in `LoginModal`.

Reply:
What would be the purpose of calling getLoginUrls again? I can look at the tests but getting rid of this second call didn't break any of the signin flows. 

We already have a URL and open the modal with that URL in this method so I'm not sure what the purpose of the second call would be